app/backend/src/couchers/models.py

```diff
@@ -184,11 +185,15 @@ class User(Base):
 
     @hybrid_property
     def is_jailed(self):
-        return self.accepted_tos < 1 or self.is_missing_location
+        return (self.accepted_tos < 1) | self.is_missing_location
 
-    @property
+    @hybrid_property
     def is_missing_location(self):
-        return not self.geom or not self.geom_radius
+        return (self.geom == None) | (self.geom_radius == None)
+
+    @hybrid_property
+    def is_visible(self):
+        return ~(self.is_banned | self.is_deleted | self.is_jailed)
```

Comment:
Jailed users should be visible

Reply:
This was the original spec you gave me. That being said, I agree. I've also come to think that "is_visible" might not be nuanced enough. See my spreadsheet here:

https://docs.google.com/spreadsheets/d/1Y1e-QvbXVE_nrvJyvdm-BVOqzEV-_519qTv9vNGk4DY/edit?usp=sharing

Reply:
Yeah, I had a look at your spreadsheet earlier and it looked good. Let's go with how it's there?